### PR TITLE
{bio}[foss/2019b,intel/2019b] OpenMM v7.4.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/62ecb8cd640b5e2f99dce48d6b987aa4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2133633949ccff8dcd6699c567728c85 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/010d64766f19582c59aa4685710f5167 for a full test report.

File: easybuild/easyconfigs/o/OpenMM/OpenMM-7.4.1_fix-Doxygen.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
+fix build errors when using Doxygen 1.8.16 like:
```
Comment (smoors) [inline easybuild/easyconfigs/o/OpenMM/OpenMM-7.4.1_fix-Doxygen.patch]: @boegel add author?

Comment (boegel) [reply easybuild/easyconfigs/o/OpenMM/OpenMM-7.4.1_fix-Doxygen.patch]: @smoors I didn't create the patch, I basically just pulled it from the pull request I reference to below...

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in this PR)
node365.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fe76fbc52a2e78e262d0c9589cd30339 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**